glean-core/ios/GleanTests/Metrics/EventMetricTests.swift

```diff
@@ -78,15 +99,20 @@ class EventMetricTypeTests: XCTestCase {
 
         XCTAssertFalse(metric.testHasValue())
 
-        metric.record(extra: [.objectId: "buttonA", .other: "foo"])
+        // Newer API
+        metric.record(ClickExtras(objectId: "buttonA", other: "foo"))
+        // Some extra keys can be left undefined.
+        metric.record(ClickExtras(objectId: "buttonA"))
 
         /* SKIPPED: resetting system clock to return fixed time value */
 
-        metric.record(extra: [.objectId: "buttonB", .other: "bar"])
+        // Old API, this is available only because we manually implemented the enum.
```

Comment:
_Is_ this the old API though?  It looks identical to the "new" API called just above on line 103...